lib/lexer_mdx.mll

```diff
@@ -16,17 +16,21 @@ let labels l =
     failwith msg
 }
 
-let eol = '\n' | eof
-let ws = ' ' | '\t'
+let eol = '\n' | '\r' '\n' | eof
+let ws = [' ' '\t']
```

Comment:
Is this required for the parsing under windows? I'm okay with this change but it might be worth extracting from this PR (or commit at least) if it's not related!

Reply:
The `ws` regex didn't change, only the syntax used to write it. I can't remember why I made this change but I think this is the right syntax to use.
The `eol` change is the goal of this PR.